### PR TITLE
Namespace pkgs support

### PR DESCRIPTION
**Pull request by thundergolfer**
### Description 

Work done to fix problems experienced in https://github.com/Canva/data-science/pull/3614/files (link private to Canva ppl). 

I think this needs an integration test, but I've at least checked this works with the above and it does 🎉.

### CODE REVIEW

File: src/extract_wheels.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 import subprocess
 import sys
 
+
+from . import namespace_pkgs
 from .wheel import Wheel
```
Comment (thundergolfer) [inline src/extract_wheels.py:9]: These relative imports are a bit dodgy. Can we get rid of them?

File: src/tests/test_namespace_pkgs.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
+import os
+import pathlib
+import unittest
```
Comment (thundergolfer) [inline src/tests/test_namespace_pkgs.py:3]: Could've used `pytest`, but this works fine and keeps us at zero `pip`  dependencies 👍.

**Comment by thundergolfer** [comment]
cc @groodt

File: src/extract_wheels.py (hunk 1 above)
Comment (dillon-giacoppo) [reply src/extract_wheels.py:9]: Yep, they are a remnant from old code. They can be replaced with `src.`

**Review by dillon-giacoppo — APPROVED** [review]
LGTM

**Pull request merged**